zip-0226.rst

```diff
@@ -320,10 +320,12 @@ For brevity, we omit the descriptions from the table below unless they differ fr
 |``852 * nActionsOrchard``    |``vActionsOrchard``       |``ZSAOrchardAction[nActionsOrchard]``      |A sequence of ZSA Orchard Action descriptions, encoded per           |
 |                             |                          |                                           |the `ZSA Orchard Action Description Encoding`_. **[UPDATED FOR ZSA]**|
 +-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+|``1``                        |``flagsFinalize``         |``byte``                                   |An 8-bit value representing a set of flags. Ordered from LSB to MSB: |
+|                             |                          |                                           | * ``finalize``                                                      |
+|                             |                          |                                           | * The remaining bits are set to ``0``.                              |
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
 |``1``                        |``flagsOrchard``          |``byte``                                   |                                                                     |
```

Comment:
think we need to remove `flagsOrchard`

Reply:
`flagsOrchard` contains the `enableSpendsOrchard` and `enableOutputsOrchard` booleans, that is still present in the Orchard part of the bundle. 

I have included all entries, even ones that do not change from earlier to our protocol -- because it is necessary to show where our additions and changes fit into the whole structure.

Reply:
isn't  **Orchard Transaction Fields** indicate that we don't want to repeat Orchard field?

Reply:
Oh, you're right, this is the transfer portion, and that is just transfer between arbitrary users. Finalization will only be by the issuer, and done during issuance. A `flagsIssuance` byte is already there in the issuance bundle. So this entry is not needed, will remove it.

Reply:
Done.